### DISCUSSION TITLE
How to make abstract & scope clearer / how not to spook people

### DISCUSSION THREAD

**Discussion by bumblefudge** [decentralized-identity/waci-didcomm#20] [Q&A]
In today's interop call, some questions: 
* "Is this a spec? Feels more like a **profile** for a specific purpose."
* "Does this **depend on** WACI? does someone who builds to thing assume a dependency on WACI?"
* Is issuance out of scope? This feels like it all tends towards Present Proof and is its goal
* Has anyone looked at the [OIDC for VPs Spec](https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0.html  - Sam: does this only work for JWT creds? Pam: You should be able to wrap LD VCs in a JWT VP this way, but let's research further!
) that just got accepted by its WG at OIDF?
* FAPI (Financial-grade API) group at OIDF has set up its own profile and certification based on that. is there a certification in the future here?

**Comment by JaceHensley** [comment]
I can take a crack at these questions, but anyone else please speak up and tell me so if I'm misspeaking or missing anything :)

- Yeah I think that makes sense, based on our last call and the blog post
- This is a spike that's based on WACI and WACI will take learnings and patterns from this effort
- That's the other side of the original WACI spec, WACI is just about interactions so request/verification is this profile's focus but the only thing that would really be swapped is Presentation Exchange => Credential Manifest
- I have not, sorry
- 🤷 I don't have an answer to this one, sorry

**Comment by bumblefudge** [comment]
Nice, thanks.  That's more or less how I answered on the call, but good to have it documented here and to confirm I didn't misspeak on the group's behalf.  Phew!

**Comment by bumblefudge** [comment]
Leaving the thread open in case someone other than me ends up PRing all this into the abstract/intro sections.  If no one else volunteers Monday, I'll take a crack next week.

**Comment by OR13** [comment]
@bumblefudge when can we expect a PR from you :)
